cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/AdaptTemplatesCommand.java

```diff
@@ -0,0 +1,239 @@
+package com.devonfw.cobigen.cli.commands;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Enumeration;
+import java.util.concurrent.Callable;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import net.sf.mmm.util.io.api.IoMode;
+import net.sf.mmm.util.io.api.RuntimeIoException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.devonfw.cobigen.cli.CobiGenCLI;
+import com.devonfw.cobigen.cli.constants.MessagesConstants;
+import com.devonfw.cobigen.cli.logger.CLILogger;
+import com.devonfw.cobigen.cli.utils.CobiGenUtils;
+import com.devonfw.cobigen.cli.utils.ConfigurationUtils;
+
+import ch.qos.logback.classic.Level;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+/**
+ *
+ */
+@Command(description = MessagesConstants.ADAPT_TEMPLATES_DESCRIPTION, name = "adapt-templates", aliases = { "a" },
+    mixinStandardHelpOptions = true)
+public class AdaptTemplatesCommand implements Callable<Integer> {
+
+    /**
+     * Utils class for CobiGen related operations
+     */
+    private static CobiGenUtils cobigenUtils = new CobiGenUtils();
```

Comment:
Why is this utils class hold as an object? Aren't the function of the classes static?